src/main/res/values/strings.xml

```diff
@@ -17,25 +17,25 @@
   <string name="anti_feature_non_free_dependencies">Depends on non-free applications</string>
   <string name="anti_feature_non_free_network">Promotes non-free network services</string>
   <string name="anti_feature_tracking">Tracks or reports your activity</string>
-  <string name="anti_feature_upstream_non_free">Upstream source code is not free</string>
+  <string name="anti_feature_upstream_non_free">Upstream source code is not libre</string>
```

Comment:
Although, it's recommended by some people to use "libre" instead of "free", the official name for "free software" is "free software". I'd like to keep "free" here.

Reply:
I'll grant you "free software" is the FSF way. "Free" is similarly a lost cause since people don't say "freeware" anymore.

Reply:
F-Droid uses simply "free" term in this place on website. It's also used this way on gnu.org (simple "free", without "free software"). I also think it should be wide-known enough for those who like free software what "free" means. I want to leave it as it is.

Reply:
I'll change that back and see if I can come up with something for the others. F-Droid and gnu.org are not examples to follow.